pkg/network/probe_handler.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// ProbeHeaderValue is the value used in 'K-Network-Probe'
+var ProbeHeaderValue = "probe"
```

Comment:
Does it have to be public?

Reply:
Yes, it will used on the client side in a subsequent PR.